apps/vr-tests-react-components/src/stories/Card/utils.tsx

```diff
@@ -5,7 +5,7 @@ import { Button } from '@fluentui/react-button';
 import { Open16Regular, Share16Regular } from '@fluentui/react-icons';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31607&commitId=2cf64a04176f0cbea4ab9033030c140ce385b8b1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3| Changed|

</details>


  <details><summary><b>Card Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.card templates.normal.chromium.png|157906| Changed|
|Card Converged.orientation.chromium.png|2811| Changed|
|Card Converged.appearance - Dark Mode.normal.chromium.png|0| Added|
|Card Converged.appearance - High Contrast.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>